projects/adrv9371x/a10gx/system_project.tcl

```diff
@@ -134,3 +134,4 @@ set_instance_assignment -name IO_STANDARD "1.8 V" -to ad9371_gpio[17]
 set_instance_assignment -name IO_STANDARD "1.8 V" -to ad9371_gpio[18]
 
 execute_flow -compile
+
```

Comment:
This new line is not necessary.  I think it was added with commit https://github.com/analogdevicesinc/hdl/pull/304/commits/61fec3aab40344951bec853d0cb690708a20be3e which doesn't seem to do anything useful

Reply:
Most likely a interactive rebase leftover. I'm deleting this commit. Thanks.